src/components/navbar/Navbar.jsx

```diff
@@ -19,7 +21,10 @@ export function Navbar() {
           <span className="badge__icons">
             <Link to="/wishlist">
               <i className="fa-regular fa-heart icon"> </i>
-              <span className="badge__number badge__primary"> 5 </span>
+              <span className="badge__number badge__primary">
+                {' '}
```

Comment:
You can remove this empty curly braces. This is not needed.

Reply:
Prettier was putting them, now I have removed them